security.rst

```diff
@@ -1906,7 +1905,10 @@ Another option is to create a route matching the name or URI you configured as
 
         class SecurityController extends AbstractController
         {
-            #[Route('/logout', name: 'app_logout', methods: ['GET'])]
+            #[Route([
+                'en' => '/logout',
+                'fr' => '/deconnexion'
+            ], name: 'app_logout', methods: ['GET'])]
```

Comment:
Then, let's also remove the PHP attribute example. Only truely advanced Symfony users will need to define their own logout route now. They understand how to create a route using the PHP attribute, so we can be a bit more strict to ourselves and comply with "Note that it doesn't need a controller, because it wouldn't be called anyways"

Reply:
So, does https://github.com/symfony/symfony-docs/pull/19000/commits/b904ca9c15f4b16bf83ff92d0f355340347d5467 solve this and https://github.com/symfony/symfony-docs/pull/19000#discussion_r1360404108?